src/Umbraco.Infrastructure/Media/ImageSharpImageUrlGenerator.cs

```diff
@@ -1,68 +1,88 @@
-            if (!string.IsNullOrWhiteSpace(options.AnimationProcessMode)) imageProcessorUrl.Append("&animationprocessmode=").Append(options.AnimationProcessMode);
-            if (!string.IsNullOrWhiteSpace(options.FurtherOptions)) imageProcessorUrl.Append(options.FurtherOptions);
+            if (options.Crop != null)
+            {
+                AddQueryString("cc", options.Crop.Left, options.Crop.Top, options.Crop.Right, options.Crop.Bottom);
+            }
 
-            //If furtherOptions contains a format, we need to put the quality after the format.
-            if (options.Quality.HasValue && hasFormat) imageProcessorUrl.Append("&quality=").Append(options.Quality);
-            if (!string.IsNullOrWhiteSpace(options.CacheBusterValue)) imageProcessorUrl.Append("&rnd=").Append(options.CacheBusterValue);
+            if (options.ImageCropMode.HasValue)
+            {
+                AddQueryString("rmode", options.ImageCropMode.Value.ToString().ToLowerInvariant());
+            }
 
-            return imageProcessorUrl.ToString();
-        }
+            if (options.ImageCropAnchor.HasValue)
+            {
+                AddQueryString("ranchor", options.ImageCropAnchor.Value.ToString().ToLowerInvariant());
+            }
 
-        private void AppendFocalPoint(StringBuilder imageProcessorUrl, ImageUrlGenerationOptions options)
-        {
-            imageProcessorUrl.Append("?center=");
-            imageProcessorUrl.Append(options.FocalPoint.Top.ToString(CultureInfo.InvariantCulture)).Append(",");
-            imageProcessorUrl.Append(options.FocalPoint.Left.ToString(CultureInfo.InvariantCulture));
-            imageProcessorUrl.Append("&mode=crop");
-        }
+            if (options.Width.HasValue)
+            {
+                AddQueryString("width", options.Width.Value);
+            }
 
-        private void AppendCrop(StringBuilder imageProcessorUrl, ImageUrlGenerationOptions options)
-        {
-            imageProcessorUrl.Append("?crop=");
-            imageProcessorUrl.Append(options.Crop.X1.ToString(CultureInfo.InvariantCulture)).Append(",");
-            imageProcessorUrl.Append(options.Crop.Y1.ToString(CultureInfo.InvariantCulture)).Append(",");
-            imageProcessorUrl.Append(options.Crop.X2.ToString(CultureInfo.InvariantCulture)).Append(",");
-            imageProcessorUrl.Append(options.Crop.Y2.ToString(CultureInfo.InvariantCulture));
-            imageProcessorUrl.Append("&cropmode=percentage");
+            if (options.Height.HasValue)
+            {
+                AddQueryString("height", options.Height.Value);
+            }
+
+            if (options.Quality.HasValue)
+            {
+                AddQueryString("quality", options.Quality.Value);
+            }
+
+            if (!string.IsNullOrWhiteSpace(options.FurtherOptions))
+            {
+                AppendQueryString(options.FurtherOptions.TrimStart('?', '&'));
```

Comment:
This ensures the further options is always added correctly (e.g. with a leading `?` when it's the first/only query string added) and you don't have to prefix this value with `&` anymore.